### DISCUSSION TITLE
카테캠 2기 아이디어톤 회고

### DISCUSSION THREAD

**Discussion by stopmin** [BDD-CLUB/challenge-every-week-posting#641] [5. 회고]
[게시글 바로가기](https://stopmin.tistory.com/entry/%F0%9F%92%A1%EC%B9%B4%EC%B9%B4%EC%98%A4%ED%85%8C%ED%81%AC%EC%BA%A0%ED%8D%BC%EC%8A%A4-%EC%95%84%EC%9D%B4%EB%94%94%EC%96%B4%ED%86%A4-%ED%9A%8C%EA%B3%A0-feat%EC%95%84%EC%9D%B4%EB%94%94%EC%96%B4-%EC%A3%BC%EA%B0%84)

어제 아이디어톤을 다녀왔는데욤! 그냥 나름 일기처럼 써봤습니다!!

**Comment by 1jeongg** [comment]
작년엔 여수였는데 올해가 밥도 더 맛있어 보이고 더 좋아보이네요 ㅋㅋㅋㅋㅋ

오 저도 이번에 주금공 신청했는데 괜히 반갑네요 ㅎㅎ

**Reply by stopmin** [reply]
헙스!! 그렇군욤 ㅎㅎ 
올해 밥 굿굿이었어욤 ㅎㅎ

**Comment by pykido** [comment]
아니 제 사진 뭔가요?? ㅋㅎㅋㅎ 수고하셨습니다

**Reply by stopmin** [reply]
제마음입니다 ㅎ
